diff --git a/library/alloc/src/lib.rs b/library/alloc/src/lib.rs
--- a/library/alloc/src/lib.rs
+++ b/library/alloc/src/lib.rs
@@ -269,7 +269,7 @@ pub mod vec;
 #[cfg(all(not(no_rc), not(no_sync), not(no_global_oom_handling)))]
 pub mod wtf8;
 
 #[doc(hidden)]
-#[unstable(feature = "liballoc_internals", issue = "none", reason = "implementation detail")]
+#[unstable(feature = "alloc_export", issue = "none")]
 pub mod __export {
     pub use core::format_args;
     pub use core::hint::must_use;
